.github/workflows/update-files.yml

```
@@ -0,0 +1,159 @@
+name: Automated Valkey Bundle Release
+
+on:
+  repository_dispatch:
+    types: [valkey-release, json-release, bloom-release, search-release, ldap-release]
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to be released'
+        required: true
+      component:
+        description: 'Valkey Bundle component to be released'
+        required: true
+        type: choice
+        options: [valkey, json, bloom, search, ldap]
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  update-files:
+    name: Update Files and Create Pull Request
+    runs-on: ubuntu-latest
+    concurrency:
+      group: valkey-bundle-updates
+      cancel-in-progress: false
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Check for an existing pull request branch
+        id: check-branch
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          BRANCH_NAME="valkey-bundle-update"
+          echo "branch-name=$BRANCH_NAME" >> $GITHUB_OUTPUT
+          
+          git config user.name "github-actions[bot]"
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+          
+          if git ls-remote --exit-code --heads origin $BRANCH_NAME; then
+            echo "branch-exists=true" >> $GITHUB_OUTPUT
+            echo "Checking out existing branch"
+            git fetch origin $BRANCH_NAME
+            git checkout $BRANCH_NAME
+            git pull origin $BRANCH_NAME
+          else
+            echo "branch-exists=false" >> $GITHUB_OUTPUT
+            echo "Staying on mainline for new branch creation"
+          fi
+
+      - name: Process inputs
+        id: process-inputs
+        run: |
+          if [[ "${{ github.event_name }}" == "repository_dispatch" ]]; then
+            # From "repository_dispatch" trigger
+            echo "version=${{ github.event.client_payload.version }}" >> $GITHUB_OUTPUT
+            echo "module=${{ github.event.client_payload.module }}" >> $GITHUB_OUTPUT
```

Can we change the `module` to `component`
```suggestion
            echo "component=${{ github.event.client_payload.component }}" >> $GITHUB_OUTPUT
```